nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestFTP.java

```diff
@@ -144,6 +146,61 @@ public void basicFileUpload() throws IOException {
         // Check file was uploaded
         Assert.assertTrue(results.exists("c:\\data\\randombytes-1"));
     }
+    @Test
+    public void basicProvenanceEventTest() throws IOException {
+        TestRunner runner = TestRunners.newTestRunner(PutFTP.class);
+        runner.setValidateExpressionUsage(false);
```

Comment:
Assuming you agree with comments above, this can be removed (as well as from lines 137/138 as well)

Reply:
Agree that since replacing test with method above, no longer required. Will remove.